flux/flux.d.ts

```diff
@@ -71,21 +73,43 @@ declare module "flux" {
 declare namespace FluxUtils {
 
     import React = __React;
+    import fbEmitter = FBEmitter;
+    import immutable = Immutable;
+    
+    /**
+     * Default options to create a Container with
+     * 
+     * @interface RealOptions
+     */
+    interface RealOptions {
+        /**
+         * Default value: true
+         * 
+         * @type {boolean}
+         */
+        pure: boolean;
+        /**
+         * Default value: false
+         * 
+         * @type {boolean}
+         */
+        withProps: boolean;
+    }
+    
     export class Container {
         constructor();
         /**
         * Create is used to transform a react class into a container
         * that updates its state when relevant stores change.
         * The provided base class must have static methods getStores() and calculateState().
         */
-        static create(base: React.ComponentClass<any>, options?: any): React.ComponentClass<any>;
+        static create<P, V extends {}>(base: React.ComponentClass<P>, options?: V & RealOptions): React.ComponentClass<P>;
```

Comment:
`Options` have only `pure` and `withProps` available variables. Why you added generic `V`?